### DISCUSSION TITLE
✅ 2FA with SFTP

### DISCUSSION THREAD

**Discussion by cotchon** [isontheline/pro.webssh.net#173] [General]
Hello,
You removed the feature on release 13.10 - july 2017 because 2FA never worked with SFTP.
Would you consider to try again to implement this great feature ?
It seems to work but I don't know what was the problem you had ...
https://help.thorntech.com/docs/sftp-gateway-classic/two-factor-authentication-with-google-authenticator/
Thank you very much

**Comment by isontheline** [comment]
Hello @cotchon 👋

Of course I can consider to check why it didn't work in past.

I'm creating an issue : https://github.com/isontheline/pro.webssh.net/issues/174
I don't have any ETA but I will check it asap.

Have a nice day ☀️

**Reply by cotchon** [reply]
Very nice, thank you :-)

**Reply by isontheline** [reply]
I've updated https://github.com/isontheline/pro.webssh.net/issues/174

Could you check it?

**Comment by isontheline** [comment]
Should be done on WebSSH 15.0 Beta Release
https://testflight.apple.com/join/QSrBK59z
